vizro-core/changelog.d/20240208_105116_huong_li_nguyen_mobile.md

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
@antonymilne - I've had this previously in:

    ```
    self.dash = dash.Dash(**kwargs, use_pages=True, pages_folder="", title="Vizro",
                              meta_tags=[{'name': 'viewport','content': 'width=device-width, initial-scale=1.0'}]
                              )
```

For context:
- https://community.plotly.com/t/how-to-get-a-responsive-layout/18029/5
- https://stackoverflow.com/questions/22777734/what-is-initial-scale-user-scalable-minimum-scale-maximum-scale-attribute-in

However, it didn't seem to affect anything visually, so I've removed it for now. But we might want to add it back in if we go for Bootstrap.

Reply:
@AnnMarieW - do you know whether this is important to include now already? 🤔

Reply:
If you are referring to:
```
meta_tags=[{'name': 'viewport','content': 'width=device-width, initial-scale=1.0'}]
```
That's the default in Dash since 2.5.0 :slightly_smiling_face:

Reply:
Ah, perfect - that's good to know! 

@AnnMarieW If you have any improvement suggestions on this PR - happy to wait for your review as well!
Especially when it comes to optimizing the Plotly charts further or any other ideas you might have :)